### ISSUE TITLE
Can´t set $translate_excluded_fields from config.yml

### ISSUE THREAD

**Issue by sebastiand** [silverstripe/silverstripe-translatable#232]
There is a private static variable in `Translatable.php` called [`$translate_excluded_fields`](https://github.com/silverstripe/silverstripe-translatable/blob/master/code/model/Translatable.php#L215-L228).
This variable is marked with `@config`, so I guess it can be set by YAML config.

I added the following lines to my `mysite/_config/config.yml` (`ChildPages` being the field that should be excluded from translation):
```yml
Translatable:
  translate_excluded_fields:
    - ChildPages
```

After running a /dev/build?flush=1 and accessing the CMS the field still gets translated. So obviously Translatable isn´t picking up the config.
When adding `ChildPages` directly to the `$translate_excluded_fields` variable in class `Translatable.php` everything works as expected, and `ChildPages` won´t get translated in the CMS:
```php
	/**
	 * Exclude these fields from translation
	 *
	 * @var array
	 * @config
	 */
	private static $translate_excluded_fields = array(
		'ViewerGroups',
		'EditorGroups',
		'CanViewType',
		'CanEditType',
		'NewTransLang',
		'createtranslation',
		'ChildPages'
	);
```

**Comment by tractorcow** [comment]
Try putting the config on the object getting translated.

```yaml
SiteTree:
  translate_excluded_fields:
    - ChildPages
```

**Comment by sebastiand** [comment]
Perfect! That works.

Shouldn´t we add that to the docs after [Translating Custom Properties] (https://github.com/silverstripe/silverstripe-translatable/blob/master/docs/en/index.md#translating-custom-properties) (where usage of `applyTranslatableFieldsUpdate()` is shown)?
Maybe in a new section "Excluding Fields from Translation"?

My suggestion:

### Excluding Fields from Translation

If you call `applyTranslatableFieldsUpdate()` on your form in `getCMSFields()` all fields will be translated by default (Note: [Some fields are excluded by default] (https://github.com/silverstripe/silverstripe-translatable/blob/master/code/model/Translatable.php#L215-L228)).
You can exclude fields from translation by setting the `$translate_excluded_fields` on the object getting translated.

This can be done via `mysite/config/_config.yml` or by setting the variable directly in your object.

If we wanted to exclude our custom `AdditionalProperty` in the above example from being translated, we could do this via `mysite/config/_config.yml`:

```yml
Page:
  translate_excluded_fields:
    - AdditionalProperty
```

or directly in our `Page` class:

    :::php
    class Page extends SiteTree {
    
        private static $db = array(
            'AdditionalProperty' => 'Text', 
        );
    
        private static $translate_excluded_fields = array(
            'AdditionalProperty'
        );
    
        function getCMSFields() {
            $fields = parent::getCMSFields();
    
            // Add fields as usual
            $additionalField = new TextField('AdditionalProperty');
            $fields->addFieldToTab('Root.Main', $additionalField);
    
            // Apply Translatable modifications
            $this->applyTranslatableFieldsUpdate($fields, 'updateCMSFields');
    
            return $fields;
        }
    }
    :::

**Comment by tractorcow** [comment]
Sure, that would be a good Pr. Do you want to update the docs and submit one for us?

Careful with linking from docs to code; Code changes over time, so docs can often become stale quickly. It would be better to actually embed the code you are linking to for the user's benefit.

**Comment by sebastiand** [comment]
I´ve added a pull request against master: https://github.com/silverstripe/silverstripe-translatable/pull/233
All linked code has gone inline.

Should we also add that to the 2.1 branch?

**Comment by tractorcow** [comment]
Fixed with #233. Closing this issue too since the problem is solved. ;D

**Comment by webspilka** [comment]
I have problem with $translate_excluded_fields
in 
"silverstripe/cms": "3.4.1",
"silverstripe/translatable": "^2.2.1",

my code is 
```php
class FitoEntry extends Page {
  private static $db = array(
    OrderStatus' => 'Boolean(1)'
  );

  function getCMSFields() {
    $fields = parent::getCMSFields();
    $fields->addFieldToTab('Root.Main', new CheckboxField('OrderStatus', "Order avaiable ?"));

    // Apply Translatable modifications
    $this->applyTranslatableFieldsUpdate($fields, 'updateCMSFields');

    return $fields;
  }

}
```
I thik after this I'll get synchronized (identical) values in all languages, but unfortunately it is not so.
But in my case, the admin in secondary languade near field 'OrderStatus' just vanished inscription
(Original value: ...)
But I can set different value for primary and secondary Languades.
I'm doing something wrong or synchronization between languages are not available?
